src/lib/fetch-api.ts

```diff
@@ -77,19 +82,19 @@ const fetchAPI: FetchApiFuncType = async (query, options) => {
 
 		(Array.isArray(errors) ? errors : [errors]).map((err) =>
 			console.error(`
-${limit}
-GraphQl API error
-${sep}
-== date:\t  ${new Date().toISOString()}
-== endpoint: ${endpoint}
-== query:
-	- name:       ${name}
-	- variables:  ${vars ?? '-'}
-	- full query: "${dedupedQuery.replace(/[\n\t\s]+/g, ' ')}"
-== error:
-${err.message}
-${limit}
-`)
+				${limit}
+				GraphQl API error
+				${sep}
+				== date:\t  ${new Date().toISOString()}
+				== endpoint: ${endpoint}
+				== query:
+					- name:       ${name}
+					- variables:  ${vars ?? '-'}
+					- full query: "${dedupedQuery.replace(/[\n\t\s]+/g, ' ')}"
+				== error:
+				${err.message}
+				${limit}
+			`)
```

Comment:
Need to start at the beggining of the line, it is important for formatting the error message in the console.

Save the file without formatting (CMD+R S)

```suggestion
${limit}
GraphQl API error
${sep}
== date:\t  ${new Date().toISOString()}
== endpoint: ${endpoint}
== query:
	- name:       ${name}
	- variables:  ${vars ?? '-'}
	- full query: "${dedupedQuery.replace(/[\n\t\s]+/g, ' ')}"
== error:
${err.message}
${limit}
`)
```